integrationTests/vm/testInitializer.go

```diff
@@ -103,20 +103,21 @@ type VMTestAccount struct {
 
 // VMTestContext -
 type VMTestContext struct {
-	TxProcessor      process.TransactionProcessor
-	ScProcessor      *smartContract.TestScProcessor
-	Accounts         state.AccountsAdapter
-	BlockchainHook   vmcommon.BlockchainHook
-	VMContainer      process.VirtualMachinesContainer
-	TxFeeHandler     process.TransactionFeeHandler
-	ShardCoordinator sharding.Coordinator
-	ScForwarder      process.IntermediateTransactionHandler
-	EconomicsData    process.EconomicsDataHandler
-	Marshalizer      marshal.Marshalizer
-	GasSchedule      core.GasScheduleNotifier
-	VMConfiguration  *config.VirtualMachineConfig
-	EpochNotifier    process.EpochNotifier
-	SCQueryService   *smartContract.SCQueryService
+	TxProcessor         process.TransactionProcessor
+	ScProcessor         *smartContract.TestScProcessor
+	Accounts            state.AccountsAdapter
+	BlockchainHook      vmcommon.BlockchainHook
+	VMContainer         process.VirtualMachinesContainer
+	TxFeeHandler        process.TransactionFeeHandler
+	ShardCoordinator    sharding.Coordinator
+	ScForwarder         process.IntermediateTransactionHandler
+	EconomicsData       process.EconomicsDataHandler
+	Marshalizer         marshal.Marshalizer
+	GasSchedule         core.GasScheduleNotifier
+	VMConfiguration     *config.VirtualMachineConfig
+	EpochNotifier       process.EpochNotifier
```

Comment:
maybe also here?

Reply:
still needed for the remaining components that still use it